rclcpp/src/rclcpp/executors/multi_threaded_executor.cpp

```diff
@@ -72,8 +72,9 @@ MultiThreadedExecutor::get_number_of_threads()
 }
 
 void
-MultiThreadedExecutor::run(size_t)
+MultiThreadedExecutor::run(size_t this_thread_number)
```

Comment:
This was a false positive from cpplint. It was interpreting this as a C-style cast...

Reply:
i see that you are just trying to fix the cpplint error. but do we actually need this argument?

Reply:
We don't, but I would argue that putting the argument here and then using `(void)this_thread_number;` (as below) is more idiomatic in this codebase.